### PR TITLE
Chore: Replaces `Gift` FSUI

### PR DESCRIPTION
**Pull request by eduardoformiga**
**Depends on  https://github.com/vtex/faststore/pull/1610**

## What's the purpose of this pull request?
- [x] Uses `Gift` from the `faststore/ui`
- [x] Adjusts `Gift` stories and related docs and styles.

## How does it work?
Everything should look as before. (Visually)

## How to test it?
You can check by adding 4 Magic mouses in the minicart.
https://sfj-1640c1e--nextjs.preview.vtex.app/apple-magic-mouse-99988216/p

### CODE REVIEW

**Review by emersonlaurentino — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -31,10 +31,10 @@
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
     "@faststore/api": "^2.0.3-alpha.0",
-    "@faststore/components": "2.0.50-alpha.0",
+    "@faststore/components": "https://pkg.csb.dev/vtex/faststore/commit/324d96ed/@faststore/components",
```
Comment (hellofanny) [inline package.json:34]: 🎗️ Update version when available

**Comment by hellofanny** [comment]
Maybe it's not related, but can you double check the SKUSelector when updating to the newest faststore package pls

image

**Comment by eduardoformiga** [comment]
> Maybe it's not related, but can you double check the SKUSelector when updating to the newest faststore package pls
> 
> image

Could you please check with the latest preview link?
https://sfj-1640c1e--nextjs.preview.vtex.app/

**Review by hellofanny — COMMENTED** [review]
LGTM! 🚀

**Review by hellofanny — APPROVED** [review]

**Pull request merged**